### ISSUE TITLE
Continue discussion from PR 11

### ISSUE THREAD

**Issue by wildpow** [babichjacob/sapper-postcss-template#13]
> I'm realizing now I needed to do the same thing in `client` or else `__sapper__/{dev,build}/client/main.[hash].css` had sourcemaps outnumbering actual CSS by ~6:1. Did you notice this?

> @babichjacob
> Yeah, I would just have turned CSS source maps off in both client and server output because I've never found the benefits but, I wasn't sure if that would have been too opinionated.
> 
> Turning them off seems to have more upsides like bundle size and a little speed, it may be better to initially have them off in both outputs and direct the user to turn them on if needed in the docs.
> 
> What do you think?

PR [11](https://github.com/babichjacob/sapper-postcss-template/pull/11) and [Conversion](https://github.com/babichjacob/sapper-postcss-template/pull/11#discussion_r421940155)

**Comment by wildpow** [comment]
One problem I see when using a boolean for the `sourcemap` value  is the console throws the error:
```shell
DevTools failed to load SourceMap: Could not load content for http://localhost:3000/client/client/main.2951653438.css.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
```

`source map has 3 possible states:
- `false`: No sourcemaps.
- `true`: Creates a `main.[chunk].css.map` file. (throwing the error)
- `inline`: (Default)

From [sapper PR 768](https://github.com/sveltejs/sapper/pull/768/commits/b4cda6f1175292c6acb10df2952f24bde7a4a9ad)

**Comment by wildpow** [comment]
JS sourcemaps are not created for production why would we need CSS ones?

Production should be:
`output: { ...config.server.output(), false },`

Client should be:
`output: { ...config.client.output(), sourcemap },`

Sourcemap var should be:
```
// If you don't use CSS sourcemaps change value to false
const sourcemap = `inline`;
 ```
and something in the readme. Let me know what you think and I'll create a PR.

**Comment by babichjacob** [comment]
> JS sourcemaps are not created for production why would we need CSS ones?

Last night I was hoping to propose that we switch the default value of `sourcemap` from `!dev` from `dev` based on this [CSS Tricks article](https://css-tricks.com/should-i-use-source-maps-in-production/). Or `dev ? "inline" : false` to combat the previously mentioned error. I don't think there's anywhere that needs this value to be a boolean, but it should be simple enough to just `!!` coerce where necessary.

It sounds like you and @ryanditjia are finding that source maps are slowing down dev significantly. For me, there is **literally no difference** *in speed*; just size. Is it that I'm not being bottlenecked by [drive speeds](https://www.samsung.com/semiconductor/minisite/ssd/product/consumer/970evo/)?

**Comment by babichjacob** [comment]
> Last night I was hoping to propose that we switch the default value of `sourcemap` from `!dev` from `dev` based on this [CSS Tricks article](https://css-tricks.com/should-i-use-source-maps-in-production/). Or `dev ? "inline" : false` to combat the previously mentioned error. 

Well, it turns out Sapper does exactly that behind the scenes: https://github.com/sveltejs/sapper/blob/master/src/config/rollup.ts#L4

**Comment by babichjacob** [comment]
Thanks for original involvement in this issue, but it's gone stale and is more or less solved by #20 and earlier commits like
b3aaa3e and 71aa809, so I'm closing it now. 🙏
